Makefile

```diff
@@ -102,7 +102,7 @@ codecov: unit
 	./hack/codecov-report.sh
 
 e2e:
-	FOCUS="\[AzureClusterReader\]|\[CustomerAdmin\]|\[EndUser\]\[Fake\]" TIMEOUT=60m ./hack/e2e.sh
+	FOCUS="\[AzureClusterReader\]|\[CustomerAdmin\]|\[EndUser\]|\[ClusterStatus\]\[Fake\]" TIMEOUT=60m ./hack/e2e.sh
```

Comment:
I assume you are going to remove `[ClusterStatus]` from the test so this should be updated too